libvcx/src/credential_def.rs

```diff
@@ -43,9 +43,43 @@ pub struct CredentialDef {
 pub struct RevocationDetails {
     pub support_revocation: Option<bool>,
     pub tails_file: Option<String>,
+    pub tails_url: Option<String>,
     pub max_creds: Option<u32>,
 }
 
+#[derive(Deserialize, Debug, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct RevocationRegistryDefinitionValue {
+    pub issuance_type: String,
+    pub max_cred_num: u32,
+    pub public_keys: serde_json::Value,
+    pub tails_hash: String,
+    pub tails_location: String
+}
+
+#[derive(Deserialize, Debug, Serialize)]
+#[serde(rename_all = "camelCase")]
```

Comment:
Remove `camelCase` here, if we actually happen to assign this serialized `RevocationRegistryDefinition ` as content of  `rev_reg_entry` in `RevocationRegistry`, we could interfere with how libindy anoncreds expects this data to look like.

Reply:
You mean content of rev_reg_def? Why do you think so? Anoncreds expects camelCase

Reply:
Oh I see. Nevermind then.